ai_powered_qa/custom_plugins/playwright_plugin.py

```diff
@@ -167,12 +226,45 @@ async def _assert_that(self, selector: str, action: str, value: str = None):
             return "Not implemented action"
         return f"Action '{action}' was successfully performed: {result_message}"
 
+    @tool
+    def scroll(self, selector: str, direction: str):
+        """
+        Scroll up or down in a selected scroll container
+
+        :param str selector: CSS selector for the scroll container
+        :param str direction: Direction to scroll in. Either 'up' or 'down'
```

Comment:
On some websites, there can be multiple scrollable regions. We mark them for the agent, and let it select the one it wants to scroll in. Then move the mouse there and turn the mouse wheel.